example/src/components/Modal/Modal.jsx

```diff
@@ -14,7 +14,74 @@ const ModalExample = () => {
           <Button title="open modal" onClick={() => setIsOpenModal1(true)}>open modal</Button>
           <Modal isOpen={isOpenModal1} hide={() => setIsOpenModal1(false)}>
             <ModalTitle icon="ri-arrow-right-fill">I am a title</ModalTitle>
-            <ModalContent>blah blah</ModalContent>
+            <ModalContent>
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+              blah blah
+              <br />
+            </ModalContent>
```

Comment:
You talk too much, man

Reply:
and yet all you can ear is blahblahblah, you never listen !!!